src/aind_data_access_api/util/docdb.py

```diff
@@ -0,0 +1,94 @@
+"""Utilities that go through the MetadataDBClient """
+
+from typing import Optional
+from aind_data_access_api.document_db import MetadataDbClient
+
+
+def get_record_from_docdb(
+    client: MetadataDbClient,
+    record_id: str,
```

Comment:
This is not resolved. The other methods still use `record_id` . The docstrings for these methods also do not match, e.g:
```
docdb_client : MongoClient
db_name : str
collection_name : str
```

Reply:
Oh I was searching in the wrong file whoops

Reply:
Btw, I used record_id because that's what's used in most of the functions in the rest of the code, so you may want to go back and change those if you want this to be consistent

Reply:
Fair! I didn't realize the other utils use record_id. Up to you if you want to keep that then. The docstrings issue is that there's extra stuff like db_name and collection_name.

Reply:
Okay I think this is sorted, you can resolve it if you think it's good

Reply:
Thanks!